Content.Server/Explosion/Components/OnUseTimerTriggerComponent.cs

```diff
@@ -44,5 +45,12 @@ public sealed class OnUseTimerTriggerComponent : Component
         /// </summary>
         [DataField("canToggleStartOnStick")]
         public bool AllowToggleStartOnStick;
+
+        /// <summary>
+        /// Have the timer trigger when mobstate changes
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("mobState")]
+        public DamageState MobState = DamageState.Alive;
     }
```

Comment:
Pretty sussy putting this here, it should just be on the separate component below. it should just be for handling a "trigger" whenever it's used. I also see the stick code infected this too so I made an issue for that to get removed.